docs/src/gis/resources/index.md

```diff
@@ -2,4 +2,7 @@
 
 Where to find internal resource collections (projects, templates, data, assets etc)
 
-External resource links
+External resource links:
+
+[GeoSeer](https://www.geoseer.net/)<br>
```

Comment:
Maybe you can add another column with a brief description of what each resource offers.

Reply:
Okay, I'll add that